tests/bsim/bluetooth/mesh/src/psa_its_emul.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/*
+ *  PSA ITS emulator over settings.
+ */
+
+#include <stdlib.h>
+
+#include <zephyr/bluetooth/mesh.h>
+#include <../library/psa_crypto_its.h>
+
+#define LOG_MODULE_NAME pts_its_emu
+
+#include <zephyr/logging/log.h>
+#include "mesh/net.h"
+#include "mesh/settings.h"
+
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL_INF);
+
+#define MAX_ITEM_LENGTH 52
```

Comment:
Where does it come from?

Reply:
Practically measured in mbedtls psa security storage implementation.

Reply:
Could you please add a comment?